themes/startup-theme-components/templates/SilverStripe/StartupThemeComponents/Elemental/Block/ImageTextBlock.ss

```diff
@@ -0,0 +1,34 @@
+<div class="image-text-block">
+    <div class="container image-text-block__container $VariantClass">
+        <div class="image-text-block__left-column">
+            <picture>
+                <source media="(min-width: 1600px)" srcset="$ImageTextBlockImage.FocusFill(882, 710).Link"/>
+                <source media="(min-width: 1200px)" srcset="$ImageTextBlockImage.FocusFill(780, 890).Link"/>
+                <source media="(min-width: 576px)" srcset="$ImageTextBlockImage.FocusFill(530, 500).Link"/>
+                <source sizes="calc(100vw)" srcset="$ImageTextBlockImage.FocusFill(575, 425).Link"/>
+                $ImageTextBlockImage.LazyLoad(false)
+            </picture>
```

Comment:
```suggestion
            <picture>
                <source media="(min-width: 992px)" srcset="$ImageTextBlockImage.ScaleWidth(885).URL"/>
                <source media="(min-width: 750px)" srcset="$ImageTextBlockImage.ScaleWidth(675).URL"/>
                <source media="(min-width: 450px)" srcset="$ImageTextBlockImage.ScaleWidth(1065).URL"/>
                <img src="$ImageTextBlockImage.ScaleWidth(615).URL" alt="$ImageTextBlockImage.Title" />
            </picture>
```
Close, but we need to support different image heights as shown below in the design:
<img width="631" alt="Screenshot 2024-10-04 at 10 26 58 AM" src="https://github.com/user-attachments/assets/5f28b3c1-3ff9-4fd8-b547-0f7da078a8a4">

**Tips:**
- I usually work with the same breakpoints that are used in the CSS
- When picking image dimensions, I like to account for for 1.5x resolution minimum
- Note for images it is `.URL`, not `.Link`
- `sizes="calc(100vw)"` is not needed in this particular type of implementation

Reply:
@DevKCode The image is getting its height cropped, when the design allows for images of different heights, please set to `scalewidth` as shown above. Please adjust breakpoints as well, as there is no point writing a media query for 1600px when the container has a max-width of 1170px.

Please always remember check the output of your code, the `img` element has a direct link to the file. So a user could upload a 50mb, and this would get directly outputted, severley impacting performance / loading times.

<img width="598" alt="Screenshot 2024-10-07 at 3 11 25 PM" src="https://github.com/user-attachments/assets/7f323c6c-3844-4a01-96f2-2827d4af9342">

Reply:
Hi @BrookeNZ I did not notice the `scalewidth`, I updated the template file now. I am restricting the Upload file size to 2MB now